cpupower: add page (#7975)
tldr-pages/tldr: pages/linux/cpupower.md

# cpupower

> Tools regarding CPU power and tuning options.
> This command is available as part of the `cpupower` package, or on Fedora as part of `kernel-tools`.

> > More information: <https://linux.die.net/man/1/cpupower>.

Reviewer: ```suggestion
> More information: <https://manned.org/cpupower>.
```

> - List CPUs:

Reviewer: When I run this command, I get
Error parsing cpu list

Arch Linux

Is this the case on other distros?

Author: I run Arch too and I've never seen that error. Alternative commands like `sudo cpupower -c all info` (and more verbose: `sudo cpupower -c all frequency-info`) exist and do list the number of each CPU.
![image](https://user-images.githubusercontent.com/64190512/163334790-78880629-9f3f-4f24-a319-1357f9e04b2b.png)

Would it be worth/should I replace `sudo cpupower --cpu cpulist` in the examples with `sudo cpupower --cpu all info`?

`sudo cpupower --cpu cpulist`

> - Get info on all cores:

Reviewer 2: We don't use abbreviations in example descriptions
 ```suggestion
- Print information about all cores:
```

> `sudo cpupower --cpu all info`

Reviewer 3: ```suggestion
`sudo cpupower --cpu {{all}} info`
```

Reviewer 3: It means user provided value. See [token syntax](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#token-syntax).

Author: Ah, I see.
![image](https://user-images.githubusercontent.com/64190512/163085539-6d660a79-9afa-4683-aacb-8d1a7d13246b.png)

- Set all CPUs to a power-saving frequency governor:

> `sudo cpupower --cpu all frequency-set --governor powersave`

Reviewer 3: ```suggestion
`sudo cpupower --cpu {{all}} frequency-set --governor {{powersave}}`
```

> - Get CPU 0's available frequency [g]overnors:

Reviewer 2: ```suggestion
- Print CPU 0's available frequency [g]overnors:
```

> `sudo cpupower --cpu 0 frequency-info g | grep "analyzing\|governors"`

Reviewer 3: ```suggestion
`sudo cpupower --cpu {{0}} frequency-info g | grep "analyzing\|governors"`
```

> - Get CPU 4's frequency from the hardware, in human-readable form:

Reviewer 2: ```suggestion
- Print CPU 4's frequency from the hardware, in human-readable form:
```

> - Print CPU 4's frequency from the hardware, in human-readable form:

Reviewer: ```suggestion
- Print CPU 4's frequency from the hardware, in a human-readable format:
```

> `sudo cpupower --cpu 4 frequency-info --hwfreq --human`

Reviewer 3: ```suggestion
`sudo cpupower --cpu {{4}} frequency-info --hwfreq --human`
```
